<ui version="4.0" >
 <class>databaseListDock</class>
 <widget class="QWidget" name="databaseListDock" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Database Entries:</string>
     </property>
     <property name="buddy" >
      <cstring>personTableWidget</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="personTableWidget" >
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>6</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Refresh</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
